lib/comment-parser.js

```diff
@@ -91,7 +91,7 @@ exports.parseExamples = function commentParser$parseExamples(parsedComments) {
   var currentCaption;
 
   for(var i = 0, len = parsedComments.length; i < len; i++) {
-    if(parsedComments[i].type === 'code') {
+    if(parsedComments[i].type === 'code' && parsedComments[i].string) {
```

Comment:
I don't really understand. It was already supported then, but missing some error handling?